src/cljam/algo/pileup.clj

```diff
@@ -84,18 +84,19 @@
   [^long ref-pos ^SAMAlignment aln]
   (let [relative-pos (- ref-pos (.pos aln))
         qual ((:quals-at-ref aln) relative-pos)
-        [base indel] ((:seqs-at-ref aln) relative-pos)]
-    (-> (PileupBase.
-         (zero? relative-pos)
-         (.mapq aln)
-         base
-         qual
-         (flag/reversed? (.flag aln))
-         (= ref-pos (.end aln))
-         (when-not (number? indel) indel)
-         (when (number? indel) indel)
-         (.qname aln))
-        (assoc :alignment aln))))
+        [base indel] ((:seqs-at-ref aln) relative-pos)
+        deletion? (number? indel)]
+    (PileupBase.
```

Comment:
Please use constructor functions instead of interop syntax.
Cf. https://github.com/bbatsov/clojure-style-guide#record-constructors

```clojure
(plpio/->PileupBase
```

Reply:
Fixed